src/main/java/com/shrek/task00003/Uniqueness.java

```diff
@@ -0,0 +1,79 @@
+package com.shrek.task00003;
+
+import javax.annotation.Nonnull;
+
+import java.util.*;
+
+/**
+ * Класс в конструкторе принимает список строк. Необхдимо реализовать 3 метода:
+ * 1. Есть ли в списке дубликаты
+ * 2. Сколько дублирующихся элементов
+ * 3. Сколько каждый элемент раз встречается
+ * <p>
+ * Написать тесты
+ */
+public class Uniqueness {
+    public List<String> elements;
+    public Set<String> elementsInSet;
+    public List<String> uniquessElements;
+
+    public Uniqueness(@Nonnull List<String> elements) {
+        this.elements = elements;
+        this.elementsInSet = new TreeSet<String>(elements);
+        this.uniquessElements = new ArrayList<String>(elementsInSet);
+    }
+
+    public int duplicateCounter() {
+        int duplicateCount = 0;
+        for (int i = 0; i < elementsInSet.size(); i++) {
+            int count = 0;
+            for (int j = 0; j < elements.size(); j++) {
+                if (elements.get(j).equals(uniquessElements.get(i))) {
+                    count++;
```

Comment:
If you meet twin is it needed to look for it again? We already know that it is duplicated. Use break